src/applications/static-pages/mhv-signin-cta/createMhvSigninCTA.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import './sass/mhv-signin-cta.scss';
+
+/**
+ * Create the MHV Signin CTA widget on a page as needed.
+ * @param {*} store the React store
+ * @param {String} widgetType the widget type
+ */
+export default async function createMhvSigninCallToAction(store, widgetType) {
+  const widgets = Array.from(
+    document.querySelectorAll(`[data-widget-type="${widgetType}"]`),
+  );
+
+  if (widgets.length) {
+    const {
+      default: MhvSigninCallToAction,
+    } = await import(/* webpackChunkName: "mhv-signin-cta" */ 'applications/static-pages/mhv-signin-cta');
+
+    widgets.forEach(el => {
+      // Grab the content that will show if no alerts.
+      const origElement = el.cloneNode(true);
+      const widgetContent = origElement.getElementsByClassName(
+        'static-widget-content',
+      );
+      ReactDOM.render(
+        <Provider store={store}>
+          <MhvSigninCallToAction
+            serviceDescription={el.getAttribute('service-description')}
```

Comment:
We probably want a test for this functionality.

Reply:
I'll look into this, but I have been unsure how I test something like this that imports and renders. There are no existing examples.

Reply:
Then, I don't understand why this component prop exists.

Reply:
Added new unit test

Reply:
I hear you, but they have added already other custom attributes. I was trying not to introduce yet another way to pass more information. I'll take a look and see how this looks.

Reply:
What I'm trying to convey is that instead of:

html: `<div service-description="a value" />`
js: `el.getAttribute('service-description')`

do the following

html: `<div data-service-description="a value" />`
js: `el.dataset.serviceDescription` or `el.getAttribute('data-service-description')`

if possible.